src/app/qgsloadstylefromdbdialog.cpp

```diff
@@ -25,10 +25,12 @@ QgsLoadStyleFromDBDialog::QgsLoadStyleFromDBDialog( QWidget *parent )
     , mSectionLimit( 0 )
 {
   setupUi( this );
-  setWindowTitle( QStringLiteral( "Load style from database" ) );
+  setWindowTitle( QStringLiteral( "Saved styles manager" ) );
```

Comment:
I don't like, but would be necessary to leave reference to DB saved styles, otherwise can be confusing if style is saved on file system.
options can be:
- DB Saved styles manager
- Manage DB save styles
- DB Saved styles
- ony other